# enable specific real-time checks for post processing good to go
# custom check_enable_t
	bool position  # 0 = disable, 1 = enable
	bool velocity  # 0 = disable, 1 = enable
	bool attitude  # 0 = disable, 1 = enable
